content/blog/cruxbox/2020-08-06_gsoc_2020_project_xfs_project_update.md

```diff
@@ -0,0 +1,46 @@
++++
+type = "blog"
+title = "GSoC 2020 Project Update: XFS"
+author = "CruxBox"
+date = "2020-11-08 17:58:00+05:30"
+tags = ["haiku", "software", "gsoc", "gsoc2020", "file system"]
++++
+
+Hello there!
+There is quite a lot to share about the work done this month.
+---
+
+Work done so far:
+You can check for the patches here: https://review.haiku-os.org/q/owner:shubhambhagat111%2540yahoo.com
```

Comment:
To force a linebreak, you have to add a couple of spaces at the end of the line.

Reply:
Ack

Reply:
For archiving purposes, it would be better to tag all the commits with the gsoc2020 tag, and link to an URL that filters on this tag. If you continue contributing to Haiku, in a few years this URL will point to a list full of other unrelated things :)